packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```
@@ -67,16 +67,18 @@ class ConfigScreenComponent extends React.Component<any, any> {
 	private async checkSyncConfig_() {
-		if (this.state.settings['sync.target'] === SyncTargetRegistry.nameToId('joplinCloud')) {
-			const isAuthenticated = await reg.syncTarget().isAuthenticated();
-			if (!isAuthenticated) {
-				return this.props.dispatch({
-					type: 'NAV_GO',
-					routeName: 'JoplinCloudLogin',
-				});
-			}
-		}
-		await shared.checkSyncConfig(this, this.state.settings);
+    	const syncTarget = reg.syncTarget();
```

ESLint

[error] Mixed spaces and tabs.

(no-mixed-spaces-and-tabs)